vuepress/docs/next/tutorials/getting-started/gke-install.md

```diff
@@ -182,52 +157,50 @@ spec:
           value: https://{HOSTNAME}/entando-de-app/
 ```
 
-2. In App Builder under `Pages → Settings` set `Base URL` to `Static`
+2. In your App Builder, go to `Pages` → `Settings` and set `Base URL` to `Static`
 
 ## Appendix A: Configuring Clustered Storage
 
-In order to scale an Entando Application across multiple nodes you must provide a storage class that supports
-a `ReadWriteMany` access policy. There are many ways to accomplish this including using dedicated storage providers
-like GlusterFS or others.
+In order to scale an Entando Application across multiple nodes, you must provide a storage class that supports
+a `ReadWriteMany` access policy, e.g. by using a dedicated storage provider like GlusterFS.
 
-The instructions below provide an example of configuring using the GCP Cloud Filestore to provide the clustered storage
-but if you have an existing enterprise clustered file solution and you can expose it as a StorageClass it is recommended to
-use and test that configuration.
+The example below uses the GCP Cloud Filestore to provide clustered storage. However, it is best practice to expose an existing enterprise clustered file solution as a StorageClass.
 
 ::: tip
-You can also scale an Entando Application without clustered storage using a `ReadWriteOnce (RWO)` policy by ensuring that the
-instances are all scheduled to the same node. This can be accomplished using taints on other nodes. Be aware of the pros and cons of scheduling
-instances to the same node. This will give you protection if the application instance itself dies or becomes unreachable and will help
-you get the most utilization of node resources. However, if the node dies or is shutdown you will have to wait for Kubernetes to reschedule the pods to a different node and your application will be down.
+You do not need clustered storage to scale an Entando Application if you schedule all instances to the same node using a `ReadWriteOnce (RWO)` policy and taints on other nodes. Be aware of the impact to node resource allocation, as well as recovery if your application fails or becomes unreachable. Note that if the node fais or is shutdown, your application will be unresponsive while Kubernetes reschedules the pods to a different node.
```

Comment:
using a `ReadWriteOnce (RWO)` policy, and taints on other nodes. 

You only need that comma if the node using RWO and taints are different.  It makes the distinction more clear.